modules/jaeger-deploy-production-streaming.adoc

```diff
@@ -0,0 +1,140 @@
+////
+This PROCEDURE module included in the following assemblies:
+- rhbjaeger-deploying.adoc
+////
+
+[id="jaeger-deploy-streaming_{context}"]
+= Deploying the Jaeger streaming strategy
+
+The `streaming` deployment strategy is intended for production environments, where long term storage of trace data is important, as well as a more scalable and highly available architecture is required.
+
+The `streaming` strategy provides a streaming capability that sits between the collector and the storage (Elasticsearch). This reduces the pressure on the storage under high load situations, and enables other trace post-processing capabilities to tap into the real time span data directly from the streaming platform (Kafka).
+
+.Prerequisites
+
+* The Elasticsearch Operator must be installed.
+* An Elasticsearch instance must be deployed.
+* The Jaeger Operator must be installed.
+* Review the instructions for how to customize the Jaeger installation.
+* An account with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with the `cluster-admin` role.
+
+. Create a new project, for example `jaeger-system`.
+
+.. Navigate to *Home* -> *Projects*.
+
+.. Click *Create Project*.
+
+.. Enter `jaeger-system` in the *Name* field.
+
+.. Click *Create*.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. If necessary, select `jaeger-system` from the Project menu.  You may have to wait a few moments for the Operators to be copied to the new project.
+
+. Click the Jaeger Operator. On the *Overview* tab, under *Provided APIs*, the Operator provides a single link.
+
+. Under *Jaeger* click *Create Instance*.
+
+. On the *Create Jaeger* page, replace the default `all-in-one` yaml text with your streaming YAML configuration, for example:
+
+.Example jaeger-streaming.yaml file
+[source,yaml]
+----
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger-streaming
+spec:
+  strategy: streaming
+  ingress:
+    openshift: {}
+    resources: {}
+    security: oauth-proxy
+  collector:
+    options:
+      kafka:
+        producer:
+          topic: jaeger-spans
+          brokers: my-cluster-kafka-brokers.kafka:9092
```

Comment:
These options are optional - so should we provide the most basic example, or explain they are optional somewhere?

Also, if the `brokers` option is not defined, then it would use self provisioned Kafka (with AMQ streams 1.4.0+).